src/frontend/src/lib/components/core/Footer.svelte

```diff
@@ -53,35 +53,44 @@
 		</div>
 
 		<div
-			class="item pointer-events-auto flex flex-row items-center justify-end gap-2 text-sm lg:max-w-48 xl:max-w-none"
+			class="item pointer-events-auto pl-6 pr-6 text-sm md:pl-0 md:pr-0 lg:max-w-48"
 			class:sm:max-w-none={$authNotSignedIn}
 			class:lg:max-w-none={$authNotSignedIn}
 			class:md:h-md:pr-4={$authNotSignedIn}
+			class:xl:max-w-none={$authNotSignedIn}
 			class:md:transition-all={$authSignedIn}
 			class:md:duration-200={$authSignedIn}
 			class:md:ease-in-out={$authSignedIn}
 			class:md:invisible={$authSignedIn}
 			class:1.5md:visible={$authSignedIn}
 			class:md:translate-x-full={$authSignedIn}
 			class:1.5md:translate-x-0={$authSignedIn}
+			class:xl:max-w-80={$authSignedIn}
```

Comment:
I now it is hard to understand what exactly is going on...
No, i can't move `xl:max-w-80` to the string definition, because this would break the behaviour of the footer on the login page.
It would look like this:
<img width="648" alt="image" src="https://github.com/user-attachments/assets/83c6823a-ef93-418d-9129-dec4be9055b1">

but it should look like this:
<img width="629" alt="image" src="https://github.com/user-attachments/assets/943d839c-6976-4daf-9cef-13d85d018b08">

Reply:
Just as an idea: maybe it would be easier to maintain this code if we do something like `${$authSignedIn ? 'sm:max-w-.. md:max-w:.. lg:max-w-... other-properties' : 'sm:max-w-.. md:max-w:.. lg:max-w-... other-properties'}`? With this format, there will be less code, and most importantly a clear difference between 2 states.

Reply:
Personally i think it is easier to understand like it is, instead of `${$authSignedIn ? 'sm:max-w-.. md:max-w:.. lg:max-w-... other-properties' : 'sm:max-w-.. md:max-w:.. lg:max-w-... other-properties'}`